app/helpers/application_helper.rb

```diff
@@ -1,4 +1,6 @@
 module ApplicationHelper
+  include CloudinaryHelper
```

Comment:
It's a little bit weird to see an helper included in another one, can I ask you why?

Reply:
Yeah, that's weird. The cloudinary helpers used in the `ApplicationHelper` are not included when called from the serializer (so there are undefined method exceptions).
`cloud_cover_url` and `article_social_image_url` are causing some trouble.
I don't know how to handle this without starting a bigger refactoring (which I don't think is a good idea at the moment) or switching back to `jbuilder`.
Please, share your ideas if you have a solution.

Reply:
I moved the code to the view objects not to include the helpers.